### PR TITLE
updated 4.2.0 upgrade doc

### PR DESCRIPTION
**Pull request by G33tha**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Comment by G33tha** [comment]
@Sunitashelar Please merge

File: developer-docs/upgrade/index.md [hunk 1]
```diff
@@ -17,27 +17,57 @@ This page details out the jobs required to be run as part of the upgrade from Su
 |-------------|------------|--------|
 |dock_private_ingressgateway_ip|Knowledge Platform FlinkJobs|Update in KP/common.yml<br/>Add dock env private ingress ip|
 |learning_content_drive_apiKey||Update in Core/common.yml<br/>[Create the Key](https://support.google.com/googleapi/answer/6158862?hl=en) and Add in Sunbird and VDN|
+|metadata2_redis_host|KP Flink|Same value as it is in DP common.yaml<br/>In KP hosts.yaml, add redis-metadata2, copy this from DP hosts.yaml|
+|crypto_encryption_key_external|Portal|generate encryption key using [link](https://www.allkeysgenerator.com/Random/Security-Encryption-Key-Generator.aspx)|
+|adminutil__portal_keyprefix: "portalv2_key"|adminutil|add in core/common.yml|
```
Comment (keshavprasadms) [inline developer-docs/upgrade/index.md:22]: @G33tha The implementation has changed in 4.3.0. So these are not relavent. Please remove these - Line # 22 to Line # 47.

File: developer-docs/upgrade/index.md [hunk 2]
```diff
@@ -17,27 +17,57 @@ This page details out the jobs required to be run as part of the upgrade from Su
...
+|Build/Core/Taxonomy|release-4.2.0_RC9|Deploy/Kubernetes/Taxonomy|release-4.2.0_RC3||
 |||Deploy/Kubernetes/UploadSchemas|release-4.2.0_RC5||
-|||Deploy/Kubernetes/Keycloak|release-4.2.0_RC2|Redeploy same artifact|
-|||Deploy/Kubernetes/Nginx-private-ingress|release-4.2.0_RC2|update the variables as per - [link](https://github.com/project-sunbird/sunbird-devops/blob/release-4.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L231-L247)|
-|||Deploy/Kubernetes/Monitoring||Create an oauth cred from google cloud account.<br/> Give redirect url as https://{domain_name}/oauth3/callback<br/>update the variables as per - [link](https://github.com/project-sunbird/sunbird-devops/blob/release-4.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L231-L247)|
+|||Deploy/Kubernetes/Keycloak|release-4.2.0_RC3|Redeploy same artifact|
+|||Deploy/Kubernetes/Nginx-private-ingress|release-4.2.0_RC3|update the variables as per - [link](https://github.com/project-sunbird/sunbird-devops/blob/release-4.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L231-L247)|
+|||OnboardConsumer|release-4.2.0_RC3||
+|||KongJWTAdminUtil|release-4.2.0_RC3||
```
Comment (keshavprasadms) [inline developer-docs/upgrade/index.md:69]: This is not required. Please remove

File: developer-docs/upgrade/index.md [hunk 3]
```diff
@@ -17,27 +17,57 @@ This page details out the jobs required to be run as part of the upgrade from Su
...
+|Build/Core/Search|release-4.2.0_RC9|Deploy/Kubernetes/Search|release-4.2.0_RC3||
+|Build/Core/Taxonomy|release-4.2.0_RC9|Deploy/Kubernetes/Taxonomy|release-4.2.0_RC3||
 |||Deploy/Kubernetes/UploadSchemas|release-4.2.0_RC5||
-|||Deploy/Kubernetes/Keycloak|release-4.2.0_RC2|Redeploy same artifact|
-|||Deploy/Kubernetes/Nginx-private-ingress|release-4.2.0_RC2|update the variables as per - [link](https://github.com/project-sunbird/sunbird-devops/blob/release-4.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L231-L247)|
-|||Deploy/Kubernetes/Monitoring||Create an oauth cred from google cloud account.<br/> Give redirect url as https://{domain_name}/oauth3/callback<br/>update the variables as per - [link](https://github.com/project-sunbird/sunbird-devops/blob/release-4.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L231-L247)|
+|||Deploy/Kubernetes/Keycloak|release-4.2.0_RC3|Redeploy same artifact|
+|||Deploy/Kubernetes/Nginx-private-ingress|release-4.2.0_RC3|update the variables as per - [link](https://github.com/project-sunbird/sunbird-devops/blob/release-4.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L231-L247)|
+|||OnboardConsumer|release-4.2.0_RC3||
```
Comment (keshavprasadms) [inline developer-docs/upgrade/index.md:68]: This is not required. Please remove

Comment (keshavprasadms) [inline developer-docs/upgrade/index.md:68]: This is not required. Please remove

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Pleasae update as per comments

File: developer-docs/upgrade/index.md (hunk 1 above)
Comment (G33tha) [reply developer-docs/upgrade/index.md]: abobe variables are removed

File: developer-docs/upgrade/index.md (hunk 3 above)
Comment (G33tha) [reply developer-docs/upgrade/index.md]: this is removed

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://846.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://846.qa.docs.sunbird.org/4.2.0/developer-docs/upgrade// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**